Lunch/TodaysLunchViewController.swift

```diff
@@ -117,15 +123,44 @@ class TodaysLunchViewController: UIViewController, MKMapViewDelegate {
 
     func mapView(mapView: MKMapView!, didSelectAnnotationView view: MKAnnotationView!) {
         if view.annotation is CustomPointAnnotation {
-            if let phoneNumber = view.annotation.subtitle {
-                if phoneNumber != nil {
-                    let regex = Regex(stringLiteral: "[0-9]*")
-                    if !phoneNumber.isEmpty && phoneNumber.match(regex) {
-                        let application = UIApplication.sharedApplication()
-                        let url = NSURL(string: "telprompt://\(phoneNumber!)")
-                        application.openURL(url!)
+            var alertView = UIAlertView()
+            alertView.delegate = self
+            alertView.addButtonWithTitle("Navigate")
+            alertView.addButtonWithTitle("Call")
+            alertView.addButtonWithTitle("Cancel")
+            alertView.title = view.annotation.title!
+            alertView.show()
+            selectedAnnotation = view.annotation as? CustomPointAnnotation
+        }
+    }
+
+    func alertView(alertView: UIAlertView!, clickedButtonAtIndex buttonIndex: Int){
+        if let annotation = selectedAnnotation {
+            switch buttonIndex{
+            case 0:
+                println("navigate")
```

Comment:
I make it a point not to commit print statements. You might like [logging from breakpoints](http://www.raywenderlich.com/28289/debugging-ios-apps-in-xcode-4-5) instead, it's pretty neat :P